CODE_OF_CONDUCT.md

```diff
@@ -14,21 +14,23 @@ Under no circumstances are users to be attacked for their ideas or contributions
 
 ### PR Approval/Objection Info
 
-Heads of Staff and Maintainers have the final say on Pull Requests. While thumbsup/thumbsdown reaction ratios are generally taken into account, they do not dictate whether or not a PR will be merged.
+Headcoders (who will take into account the votes from both the Design and Balance teams) have the final say on Pull Requests. While thumbsup/thumbsdown reaction ratios are generally taken into account, they do not dictate whether or not a PR will be merged.
 
-After a twenty four hour minimum waiting period, Pull Requests can be merged once they receive approval from both a Head of Staff and a Maintainer. An exception is made for refactors and fixes, which may be merged at a Maintainer's discretion with no waiting period.
+After a twenty four hour minimum waiting period, Pull Requests can be merged once they receive approval from the relevant team. An exception is made for refactors and fixes, which may be merged by any member with commit access' discretion with no waiting period.
 
-While normally provided, Heads of Staff and Maintainers are not obligated to publicly state their objections to a Pull Request. Attacking or berating either of these roles over an objection will not be tolerated. Additionally, whining over the closure of a PR, the existence of an objection, or similar behaviour, will not be tolerated.
+While normally provided, Design and Balance team members are not obligated to publicly state their objections to a Pull Request. Attacking or berating either of these roles over an objection will not be tolerated. Additionally, whining over the closure of a PR, the existence of an objection, or similar behaviour, will not be tolerated.
+
+Headcoders may close your PR at their discretion if your PR history has little focus on improving repo maintainability (ie: making nothing but 20 balance or feature PRs). Likewise, balance PRs may be closed if the PR author has little-to-no time played on the server. This is to ensure balance changes are made by people actually in-touch with the server atmosphere.
 
 ### PR Expectations
 
 All Pull Requests are expected to be tested prior to submission. If a submitted Pull Request fails to pass CI checks, the likelihood of it being merged will be significantly lower. If you can't take the time to compile/test your Pull Request, do not expect a warm reception.
 
-It is expected that contributors discuss larger changes on the forums prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Maintainers are not responsible for contributors wasting their time creating features nobody asked for.
+It is expected that contributors discuss larger changes on the forums prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Repo staff are not responsible for contributors wasting their time creating features nobody asked for.
```

Comment:
Could you extend the platforms that changes should be discussed on in `discuss larger changes` to also include the repository discussions tab or would you prefer a forums thread be a *required* part of larger changes (perhaps due to the accessibility to the discussion by non devs?)

Reply:
Can do